src/data/markdown/translated-guides/en/01 Getting started/02 Installation/01 Troubleshooting.md

```diff
@@ -10,22 +10,49 @@ If you use such a distribution, you'll need to install them:
 
 ```bash
 sudo apt-get update && sudo apt-get install ca-certificates gnupg2 -y
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
-echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
```

Comment:
I noticed that this section is supposed to fix the missing `ca-certificates` and `gnupg2` packages problem. So we don't need to repeat the k6 installation instructions. Just the first `apt-get install` command is fine.

Reply:
I fixed it in 839e7d7a (#646).

Reply:
Once again, thanks a lot for that @imiric 🙇🏻